job-templates/kubernetes_release.json

```diff
@@ -20,7 +20,8 @@
       {
         "name": "windowspool2",
         "count": 2,
-        "vmSize": "Standard_D2_v3",
+        "vmSize": "Standard_D2s_v3",
+	"osDiskSizeGB": 128,
```

Comment:
small misalignment here, other than that LGTM

Reply:
fixed, I got a tab in by mistake